libs/cardano-ledger-binary/cardano-ledger-binary.cabal

```diff
@@ -1,6 +1,6 @@
 cabal-version: 3.0
 name:          cardano-ledger-binary
-version:       1.3.1.0
+version:       1.3.2.0
```

Comment:
One last thing that I didn't notice last time reviewing this PR is that `cardano-ledger-conway` need a lower bound on the `cardano-ledgder-biary >= 1.3.2`, since it is using the new functions